cmd/photobak/main.go

```diff
@@ -47,6 +48,13 @@ type daemon struct {
 }
 
 func startDaemon(interval time.Duration) {
+	if runtime.GOOS != "windows" {
+		// The default behaviour on SIGPIPE is to silently terminate the program which breaks clean shutdown, so ignore
+		// it because every program should check write() return code instead of crashing if some file descriptor became
+		// unavailable for writing.
+		signal.Notify(make(chan os.Signal), syscall.SIGPIPE)
```

Comment:
Unless SIGPIPE exists in Windows, I do not think this will compile on Windows... because the syscall.SIGPIPE symbol doesn't exist. Did you double check? 😇 I think a cross-compile will do the trick...